### PR TITLE
Added Dual Shock 2 Win - Benjamin Siskoo

### PR DESCRIPTION
**Pull request by BenjaminSiskoo**

### CODE REVIEW

File: gamecontrollerdb.txt [hunk 1]
```diff
@@ -80,6 +80,7 @@
 030000007d0400000840000000000000,Destroyer Tiltpad,+leftx:h0.2,+lefty:h0.4,-leftx:h0.8,-lefty:h0.1,a:b1,b:b2,dpdown:+a1,dpleft:-a0,dpright:+a0,dpup:-a1,leftshoulder:b4,rightshoulder:b5,x:b0,y:b3,platform:Windows,
 03000000791d00000103000000000000,Dual Box WII,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b10,leftshoulder:b6,lefttrigger:b4,leftx:a0,lefty:a1,rightshoulder:b7,righttrigger:b5,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Windows,
 03000000bd12000002e0000000000000,Dual USB Vibration Joystick,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,leftshoulder:b6,leftstick:b9,lefttrigger:b4,leftx:a0,lefty:a1,rightshoulder:b7,rightstick:b10,righttrigger:b5,rightx:a3,righty:a2,start:b11,x:b3,y:b0,platform:Windows,
+030000008f0e00000910000000000000,Dual Schock 2,platform:Windows,a:b2,b:b1,x:b3,y:b0,back:b8,start:b11,leftstick:b9,rightstick:b10,leftshoulder:b6,rightshoulder:b7,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a3,righty:a2,lefttrigger:b4,righttrigger:b5,
```
Comment (gabomdq) [inline gamecontrollerdb.txt:83]: There's a typo in the controller name.

Comment (BenjaminSiskoo) [reply gamecontrollerdb.txt]: Ok you right, fixed thanks :)

File: gamecontrollerdb.txt [hunk 2]
```diff
@@ -80,6 +80,7 @@
 030000007d0400000840000000000000,Destroyer Tiltpad,+leftx:h0.2,+lefty:h0.4,-leftx:h0.8,-lefty:h0.1,a:b1,b:b2,dpdown:+a1,dpleft:-a0,dpright:+a0,dpup:-a1,leftshoulder:b4,rightshoulder:b5,x:b0,y:b3,platform:Windows,
 03000000791d00000103000000000000,Dual Box WII,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b10,leftshoulder:b6,lefttrigger:b4,leftx:a0,lefty:a1,rightshoulder:b7,righttrigger:b5,rightx:a2,righty:a3,start:b9,x:b3,y:b0,platform:Windows,
 03000000bd12000002e0000000000000,Dual USB Vibration Joystick,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,leftshoulder:b6,leftstick:b9,lefttrigger:b4,leftx:a0,lefty:a1,rightshoulder:b7,rightstick:b10,righttrigger:b5,rightx:a3,righty:a2,start:b11,x:b3,y:b0,platform:Windows,
+030000008f0e00000910000000000000,DualShock 2,platform:Windows,a:b2,b:b1,x:b3,y:b0,back:b8,start:b11,leftstick:b9,rightstick:b10,leftshoulder:b6,rightshoulder:b7,dpup:h0.1,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,leftx:a0,lefty:a1,rightx:a3,righty:a2,lefttrigger:b4,righttrigger:b5,
```
Comment (RedDevilus) [inline gamecontrollerdb.txt:83]: ,platform:Windows has to be the end of the string.

Comment (offalynne) [reply gamecontrollerdb.txt:83]: no, it doesn’t.

Comment (RedDevilus) [reply gamecontrollerdb.txt:83]: Unfortunately, Steam outputs the platform field at the beginning, so you will need to move it to the end manually. Move platform:Windows, or platform:Mac OS X, or platform:Linux, to the end of the mapping (with the trailing comma).
Then the readme should be changed. It implied to me it should be at the end, everything except steam does it wrong. But fair enough.

Comment (offalynne) [reply gamecontrollerdb.txt:83]: https://github.com/gabomdq/SDL_GameControllerDB/pull/398

Comment (gabomdq) [reply gamecontrollerdb.txt:83]: SDL parses these fields with no expectation of order beyond the first two fields (GUID and name), so any sorting here should be fine.

**Pull request merged**